### DISCUSSION TITLE
观看港澳台番剧功能失效

### DISCUSSION THREAD

**Discussion by ManyullynSword** [ywmoyue/biliuwp-lite#766] [bug]
### Bug 描述

同油猴插件，通过漫游服务器观看港澳台番剧的功能失效了。

### 操作步骤

```plain text
1.打开应用
2.设置好可用的漫游服务器
3.随便搜索
4.港澳台结果出不来
```

### 期望结果

可以像以前那样还能正常观看。

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2

### 设备类型

台式机

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
哪个漫游服务器，我测了ayo.moe是漫游X和bililite都有效的

**Reply by ManyullynSword** [reply]
我这边看不了番剧，说是被属鼠制裁了，或者无数据返回。

**Reply by ywmoyue** [reply]
所以用的哪个漫游服务器，或者发一下日志看看

**Comment by ManyullynSword** [comment]
[Uploading 20240819.log…]()

**Reply by ManyullynSword** [reply]
这是日志。

**Reply by ywmoyue** [reply]
这个日志无法下载，markdown链接并没有指向一个文件地址

请问你用相同漫游服务器用其他客户端是能播放的吗

**Reply by ManyullynSword** [reply]
可以正常play在哔哩漫游X客户端。

**Comment by ManyullynSword** [comment]
bili.ayo.moe

**Reply by ManyullynSword** [reply]
这是解析服务器

**Comment by ywmoyue** [comment]
我这几天有空就试试能不能适配这个代理：https://github.com/ywmoyue/biliuwp-lite/discussions/790

如果能适配的话应该能修复这个问题

**Comment by ywmoyue** [comment]
试试这个版本有效吗，https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.2.302

**Reply by ManyullynSword** [reply]
仍然给我制裁了

**Reply by ManyullynSword** [reply]
[Uploading 20240831.txt…]()

**Comment by ywmoyue** [comment]
看日志搜索请求发不出去，播放请求是200响应，可能这个号被服务器拉黑了？但你说漫游x上面正常 🤔

**Reply by ManyullynSword** [reply]
可是漫游安卓端确实可以正常使用的啊

**Reply by ywmoyue** [reply]
所以我这边没有其他方式能定位这个问题了，只能建议你尝试下这个自行部署的项目能否使用
https://github.com/ywmoyue/biliuwp-lite/discussions/790#discussioncomment-10460972
